src/ui/flutter_app/lib/l10n/intl_en.arb

```diff
@@ -44,13 +44,15 @@
     "@testViaLAN": {
         "description": "Test Via LAN"
     },
-    "move": "Move",
-    "@move": {
-        "description": "Move"
-    },
-    "moves": " Moves",
-    "@moves": {
-        "description": " Moves"
+    "move_number": "{count,plural, =0{Move}=1{{count} Move}other{{count} Moves}}",
```

Comment:
In this case Intl will make the pluralization for us. I used 0 to indicate that we want it without any count variable (this might need to be changed. Forgot that after testing a bit)